scripts/upload/app-store-connect.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Pushes iOS and macOS builds to App Store Connect
+
+set -euo pipefail
+
+# xcrun altool requires private keys to be files in a specific naming format
+API_PRIVATE_KEYS_DIR=${API_PRIVATE_KEYS_DIR:-$RUNNER_TEMP}
+PRIVATE_KEY_PATH="$API_PRIVATE_KEYS_DIR/AuthKey_$API_KEY_ID.p8"
+echo -n "$API_KEY" | base64 --decode -o "$PRIVATE_KEY_PATH"
```

Comment:
Slight change here to use a base64-encoded version of the API key. It is already in a PEM format, but the downstream build-rust.sh script was barking on its contained newlines, so this fixes that.

Reply:
Could also reuse that utility decode function used in `lib.sh` when building. Not sure if it is worth consolidating though.

Reply:
👍